lib/order.rb

```diff
@@ -0,0 +1,39 @@
+# Allows takeaway to create an order, add to basket, display basket and display total price" 
+require_relative 'menu'
+
+class Order
+  attr_reader :basket_summary , :total
+
+  def initialize
+    @basket_summary = Hash.new(0)
+    @total = 0
+  end
+
+  def add(dish, quantity = 1)
+    @basket_summary[dish] += quantity
+  end
+
+  def display_total
+    price_calc
+    "Your order total is £#{@total}"
+  end
+
+  # private
+  def price_calc
```

Comment:
I think you were right in the first place - this method should probably be private. It seems as though a user would interact with the `display_total` method rather than this one, so I don't think it needs to be exposed.

My guess is that you removed the privacy so you could test it? Hopefully what we spoke about in kickoff this morning has helped you understand the correct approach here 🙂